js_modules/dagster-ui/packages/ui-core/src/asset-graph/Utils.tsx

```diff
@@ -261,3 +261,21 @@ export const itemWithAssetKey = (key: {path: string[]}) => {
   const token = tokenForAssetKey(key);
   return (asset: {assetKey: {path: string[]}}) => tokenForAssetKey(asset.assetKey) === token;
 };
+
+export function walkTreeUpwards(
```

Comment:
Hmm lmk what you need here, it's pretty likely we have a helper for this and it just uses a different name!